lib/google_geocoding_api.ex

```diff
@@ -3,8 +3,8 @@ defmodule GoogleGeocodingApi do
   Provides functions to interact with Google Geocoding API.
   """
 
-  def all_info(address) do
-    result = Poison.decode!(make_request(address).body)
+  def all_info(address, opts \\ []) do
```

Comment:
👌 nice, makes it easy to add other optional arguments down the line. Good thinking